docs/source/_templates/layout.html

```diff
@@ -0,0 +1,5 @@
+{% extends "!layout.html" %}
+{% block extrahead %}
+    {{ super() }}
+    <meta name="robots" content="noindex, nofollow">
```

Comment:
We don't want to do this in all cases right? Only when the version is old

Reply:
yes, it's possible to modify it to handle that, I'm just trying to understand, is it in reality possible to rebuild our old versions, what do you think? And currently I'm also thinking maybe that `noindex` is not the right solution at all:
https://github.com/kedro-org/kedro/issues/3741#issuecomment-2490590143

Reply:
> is it in reality possible to rebuild our old docs versions, what do you think?

We did it to address a security issue https://github.com/kedro-org/kedro/issues/3864 less than ideal, I know.

However, there's another way of doing it that doesn't require rebuilding: https://github.com/readthedocs/readthedocs.org/issues/10648#issuecomment-2485945574 we should try that first.

In any case, I think the priority for this PR should be to avoid blocking the subprojects.

> For the current release, I propose moving forward only with a manual update to robots.txt and sitemap.xml that will solve P0.

👍🏼